components/PictureGrid.js

```diff
@@ -1,20 +1,24 @@
 import React, { PropTypes } from 'react';
+import Picture from '../components/Picture';
 
-const PictureGrid = () => {
-  const pictures = [{
-    url: 'blah',
-  }];
-
+const PictureGrid = (props) => {
+  const pictureItems = props.pictures.map((picture) =>
```

Comment:
When you're doing an arrow return that spans multiple lines, put parentheses around the return object.

```js
props.pictures.map((picture) => (
  <Picture
    ...
  />
));
```